app/models/order.rb

```diff
@@ -6,4 +6,13 @@ class Order < ApplicationRecord
   has_many :items, through: :order_items
 
   enum status: ['pending', 'packaged', 'shipped', 'cancelled']
+
+  def total_quantity
+    self.items.sum(:quantity)
```

Comment:
I think we want to sum the `order_items.quantity`

The way you have it sums the `items.quantity` which is the _stock available from the merchant._

Reply:
Why did we name both of them quantity 🤷‍♂️ haha